chainer/training/triggers/minmax_value_trigger.py

```diff
@@ -0,0 +1,89 @@
+from chainer import reporter
+from chainer.training.trigger import IntervalTrigger
+
+
+class BestValueTrigger(object):
+
+    """Trigger invoked when specific value becomes best.
+
+    Args:
+        key (str): Key of value.
+        compare (function): Compare function which takes current best value and
+        new value and returns whether new value is better than current best.
+
+    """
+    def __init__(self, key, compare):
+        self._key = key
+        self._min_value = None
+        self._interval_trigger = IntervalTrigger(1, 'epoch')
```

Comment:
It's a nice idea.
I've fixed `BestValueTrigger` to accept `trigger` option.
7be533b0fd879edd093f765a6e4ed75b659a9079